assets/panel/debugger.properties

```diff
@@ -555,6 +555,9 @@ outline.header=Outline
 # LOCALIZATION NOTE (outline.noFunctions): Outline text when there are no functions to display
 outline.noFunctions=No functions
 
+# LOCALIZATION NOTE (outline.noFileSelected): Outline text when there are no functions to display
+outline.noFileSelected=No file selected
```

Comment:
Should this be "… no file**s**"?

Reply:
Since we can only ever select one file I'd keep the singular here? Same as in https://searchfox.org/mozilla-central/source/dom/locales/en-US/chrome/layout/HtmlForm.properties#17-22

Reply:
I'm not a native speaker, but I usually use a plural after "no". I guess both singular and plural work though.

Reply:
Yes I was unsure about that too. googling + looking through the codebase did not really help getting a definitive answer, so I think leaving as is is fine.